### DISCUSSION TITLE
Packetsender on Raspberry Pi

### DISCUSSION THREAD

**Discussion by fudge-but** [dannagle/PacketSender#331] [General]
Hi

I there a way to it working on a Raspberry Pi Model 4 B?
I bought the Raspberry thinking i could just install it there but when i use SNAP to install it says there are not package for ARM64 but there is for AMD64.

**Comment by dannagle** [comment]
Yes, Raspberry Pi is supported if you compile and install by source. Install the Qt SDK, clone the Packet Sender repo, and then build it. 

Some info in the README here:
https://github.com/dannagle/PacketSender#building-packet-sender-for-linux

Those were written for Ubuntu. Raspi and Ubuntu are both Debian-based, so the instructions should be similar.  Note that Packet Sender has recently moved to Qt6. I don't know which Qt is available for Raspberry Pi, but Qt5 should still work.
